includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -620,32 +635,47 @@ public static function add_admin_hooks() {
 		add_filter( 'manage_edit-' . self::TAXONOMY_SLUG . '_columns', function( $old_columns ) {
 			return array(
 				'cb'               => $old_columns['cb'],
-				'error'            => __( 'Error', 'amp' ),
-				'created_date_gmt' => __( 'Created Date', 'amp' ),
+				'error'            => __( 'Error Inventory', 'amp' ),
```

Comment:
Actually I don't think this column name makes sense here. If we are going to use “Error Inventory” then this would be for the entire screen, not just for the one column. The column makes more sense as just “Error” I believe. I'll change it.